rule-packs/compliance.json

```diff
@@ -0,0 +1,14 @@
+[
+  {
+    "name": "compliance-gaps",
+    "description": "Finds open compliance gaps.",
+    "queries": [
+      {
+        "name": "gaps",
+        "query": "FIND jupiterone_compliance_gap",
+        "version": "v1"
+      }
+    ],
+    "alertLevel": "HIGH"
```

Comment:
Actually perhaps this is better as Medium?

Reply:
I'm cool with that

Reply:
done